evennia/contrib/aisystem/README.md

```diff
@@ -0,0 +1,661 @@
+## Behavior tree-based AI
+The AI this system provides is based on behavior trees, structures that evaluate
+situations and perform actions in a specified order. As of 2016, behavior trees
```

Comment:
Since you are fererring to AIObjects/AIScripts later in the tree overview without defining them, it might be a good idea to briefly explain them here. Overall, pointing out that the behavior tree is assigned to/used by such an agent is a good idea. An introductory list of features and an example of AI behavior possible with this system might also be good to put at the top, to show users why this might benefit them over a more traditional state-based AI approach.